AsmSpy.CommandLine/Program.cs

```diff
@@ -144,7 +145,11 @@ public static int Main(string[] args)
             {
                 Console.WriteLine(cpe.Message);
                 commandLineApplication.ShowHelp();
-                return 0;
+                return -1;
+            }
+            catch (Exception)
+            {
+                return -1;

```

Comment:
I know this is already merged - but completley ignoring the message might not be ideal, right?

Returning -1 is good, but earlier you could atleast see the unhandled exception on the console, now it is handled and can be nicely catched by calling tools, but you get no indication anymore on what went wrong.

Reply:
That's a good point @pfeigl. What do you think @patriksvensson?

Reply:
My take would btw. be very simple and just do `Console.Error.WriteLine(ex);`

Reply:
@mikehadlow This was a complete oversight on my end when making the change. Logging the error like @pfeigl suggest sounds like a good idea. I can submit a pull request for this tonight unless @pfeigl already have addressed this.

Reply:
@patriksvensson, that sounds great, I'd welcome a PR for this.